res/values-pt-rBR/strings.xml

```diff
@@ -1360,13 +1360,13 @@ Comandos:
     <string name="Master_Tamer">Domador Magistral</string>
     <string name="Get_ALL_Pets_To_Level_100_">Leve TODOS os mascotes ao nível 100</string>
     <string name="TAMER">DOMADOR</string>
-    <string name="Allow_Recombine">Permitir Recombinar</string>
+    <string name="Allow_Recombine">Permitir Recombinaçaõ</string>
```

Comment:
2 issues:

1. Your change contains a typo;
2. The previous text was already correct (based on english value).